packages/contracts-bedrock/src/L2/OperatorFeeVault.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Contracts
+import { FeeVault } from "src/L2/FeeVault.sol";
+
+// Libraries
+import { Types } from "src/libraries/Types.sol";
+
+// Interfaces
+import { ISemver } from "interfaces/universal/ISemver.sol";
+
+/// @custom:proxied true
+/// @custom:predeploy 0x420000000000000000000000000000000000001B
+/// @title OperatorFeeVault
+/// @notice The OperatorFeeVault accumulates the operator portion of the transaction fees.
+contract OperatorFeeVault is FeeVault, ISemver {
+    /// @notice Semantic version.
+    /// @custom:semver 1.5.0-beta.5
+    string public constant version = "1.5.0-beta.5";
+
+    /// @notice Constructs the OperatorFeeVault contract.
+    /// @param _recipient           Wallet that will receive the fees.
+    /// @param _minWithdrawalAmount Minimum balance for withdrawals.
+    /// @param _withdrawalNetwork   Network which the recipient will receive fees on.
```

Comment:
To simplify things, would you just be in favor of hardcoding in the basefeevault here as the recipient and the withdrawal network of L2 with a min withdrawal amount of 0? Anybody can deploy a new fee vault and upgrade to it. What this does is makes it so we can just use the deploy artifact when getting the upgrade tx calldata instead of us needing to fill in the arguments manually

Reply:
Yes I can do that

Reply:
Done in this commit: https://github.com/ethereum-optimism/optimism/pull/12166/commits/2f5b4d3484d3506dbda52eab997f11082aa8f458